recipes/gem3-mapper/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+#adding include and library paths in order to find bzip2
+sed -i.bak '
+    /^PATH_INCLUDE=/ s@$@ -I'$PREFIX'/include@
+    /^PATH_LIB=/ s@$@ -L'$PREFIX'/lib@
+  ' Makefile.mk.in
+
+
+./configure 
```

Comment:
./configure --prefix=$PREFIX?

Reply:
Nope, it is not enough. I also tried with CFLAGS and LDFLAGS, but they are not respected. Thanks for the suggestion though, I didn't think of it earlier.